_data-prepper/pipelines/configuration/processors/list-to-map.md

```diff
@@ -0,0 +1,303 @@
+---
+layout: default
+title: List to map processor
+parent: Processors
+grand_parent: Pipelines
+nav_order: 55
+---
+
+# list_to_map
+
+The `list-to-map` processor converts a list of objects from an event where each object contains a `key` field, into a map of target keys.
+
+## Configuration
+
+The following table describes the configuration options to generate target keys to object mappings:
```

Comment:
This language seems to disagree with the processor description above. (i.e., converts a list of objects into a map of keys).
Since the process is described above, can you generally describe what the options do for you like so?
```suggestion
The following table describes the configuration options used to generate target keys for the mappings:
```